lib/filewatch/watch.rb

```diff
@@ -207,20 +207,24 @@ def subscribe(stat_interval = 1, discover_interval = 5, &block)
     end # def subscribe
 
     private
-    def expired?(stat, watched_file)
-      file_expired?(stat) && watched_file.size == stat.size
+    def close_expired?(stat, watched_file)
+      file_close?(stat) && watched_file.size == stat.size
```

Comment:
I'l partly agree with you because the discover_ignore is also problematic.
I propose:
`file_close?` -> `file_can_close?`
`close_expired?` -> `file_closable?`
`discover_ignore` -> `file_ignorable?`


wdyt-able?